psydac/fem/basic.py

```diff
@@ -69,7 +69,7 @@ def is_product( self ):
     # Abstract interface: evaluation methods
     #---------------------------------------
     @abstractmethod
-    def eval_field( self, field, *eta ):
+    def eval_field( self, field, *eta , weights=None):
         """
         Evaluate field at location(s) eta.
```

Comment:
Please update docstring. It should explain that `weights.space == field.space`

Reply:
okay